service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml

```diff
@@ -0,0 +1,1417 @@
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "assembly_length"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.assembly_length_unambiguous",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "assembly_length_unambiguous"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.assembly_mean_coverage",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Float"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "assembly_mean_coverage"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.assembly_method",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "destination": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.dist_to_ref_indels",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "dist_to_ref_indels"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.dist_to_ref_snps",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "dist_to_ref_snps"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.ivar_trim_stats",
+                        "output_type": {
+                          "type": "array",
+                          "array_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
```

Comment:
In our sample workspace this is `Array[Map[String, String]]`. 

Since we don't want to support maps yet, and the output is set to "none", this is probably ok for now... but I wonder whether we'd be better off defining the map type and just failing if people try to use it, rather than pretending that maps are strings?

Reply:
I'll make a jira ticket for it